src/CommerceModel.BetterRetail/artifacts/OOE/BetterRetail/ExecuteSQLScriptActivity/Order/AddExternalPaymentProvider.sql

```diff
@@ -8,7 +8,6 @@ CREATE TABLE #ScopeToAdd (
 INSERT INTO #ScopeToAdd 
 VALUES 
     ('BetterRetailCanada','7E5F8183-CBF4-467B-9200-6F39D586B64A')
-    ,('BetterRetailNetherlands','D4D01053-67A1-4CF4-A194-2B65DE20A63B')
```

Comment:
This should not be removed
`,('BetterRetailEuro','D4D01053-67A1-4CF4-A194-2B65DE20A63B')`